### ISSUE TITLE
VPN not masking ip address

### ISSUE THREAD

**Issue by Valar-Morghul1s** [sgtsquiggs/docker-deluge-openvpn#16]
Hello all! Full disclosure, I'm very new to this, so I apologize in advance for any naive mistakes I may have made. I had my torrent client working well on my Synology DS218+ through docker, but wanted to switch from Transmission to Deluge since I prefer the plugins and options you get. I kept all of the OpenVPN settings the same as I'd used in haugene's transmission OpenVPN and got the webui and deluge client working, but when I tested the VPN coverage with ipmagnet, the test VPN was registered as my ACTUAL ip address, instead of one from my VPN. I'm currently using PIA for my VPN client. Any recommendations for how to fix this?

**Comment by Pompey69** [comment]
I don't think you can do IP magnet with a GUI on a container. It would run off your web browser which would not be your container's vpn
